pkg/lib/queueinformer/queueinformer_operator.go

```diff
@@ -134,6 +134,7 @@ func (o *Operator) sync(loop *QueueInformer, key string) error {
 		return err
 	}
 
+	// TODO: handle resource deletion metrics here?
```

Comment:
I'm wondering if we could [pass in a custom event handler](https://github.com/operator-framework/operator-lifecycle-manager/blob/b870d912cd4f7e3e853a347432fa44cb4874ea78/pkg/lib/queueinformer/queueinformer.go#L88) from the operator that has the resource deletion metrics embedded in the custom `cache.ResourceHandlerFuncs.Delete` function

Reply:
I thought about that. It's worth considering, but I didn't immediately gravitate towards it since I think I'd have to have that delete function be aware of all the types. Then again, the TODO location would have the same problem.

Reply:
Good point. I also just remembered that everything in the `lib` directory is stuff that we wish was in an external package.